docs-ref-services/advisor.md

```diff
@@ -28,28 +28,12 @@ With Advisor, you can:
 Install the Azure Advisor npm module
 
 ```bash
-npm install azure-arm-advisor
+npm install @azure/arm-advisor
 ```
 
-### Example
-
-This example displays the list of recommendations from Azure Advisor.
-
-```javascript
-const msRestAzure = require('ms-rest-azure');
-const advisorManagement = require('azure-arm-advisor');
-
-const subscriptionId = 'your-subscription-id';
+## Samples
 
-msRestAzure.interactiveLogin().then(credentials => {
-  const client = new advisorManagement(credentials, subscriptionId);
-  client.recommendations.list().then(recommendations => {
-    console.log('List of recommendations:');
-    console.dir(recommendations, {depth: null, colors: true});
-  });
-});
-```
+* Examples for using this module in Node.js as well as browser applications can be found in the [README for the module](https://www.npmjs.com/package/@azure/arm-advisor)
 
-## Samples
 
-Explore more [sample Node.js code](https://azure.microsoft.com/resources/samples/?platform=nodejs) you can use in your apps.
+* For more code samples that use various Azure packages, explore the [Node.JS samples](https://docs.microsoft.com/samples/browse/?languages=nodejs).

```

Comment:
this one was changed so probably both link name and link should be javascript [JavaScript samples](https://docs.microsoft.com/samples/browse/?languages=javascript)